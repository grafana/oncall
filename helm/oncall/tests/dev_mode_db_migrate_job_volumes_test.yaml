suite: dev mode - db migrate job
templates:
  - engine/job-migrate.yaml
release:
  name: oncall
tests:
  - it: doesn't add a volume and volume mount when oncall.devMode is disabled
    set:
      oncall:
        devMode: false
    asserts:
      - notExists:
          path: spec.template.spec.containers[0].volumeMounts
      - notExists:
          path: spec.template.spec.volumes
  - it: does add a volume and volume mount when oncall.devMode is enabled
    set:
      oncall:
        devMode: true
    asserts:
      - matchSnapshot:
          path: spec.template.spec.containers[0].volumeMounts
      - matchSnapshot:
          path: spec.template.spec.volumes
