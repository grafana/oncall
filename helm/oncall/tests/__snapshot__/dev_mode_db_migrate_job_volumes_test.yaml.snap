does add a volume and volume mount when oncall.devMode is enabled:
  1: |
    - mountPath: /etc/app
      name: dev-reloaded-engine
  2: |
    - hostPath:
        path: /engine
      name: dev-reloaded-engine
